tox.ini

```diff
@@ -1,15 +1,108 @@
+# Here is what you can do:
+
+# tox                    # run tests with pytest (you can pass extra arguments
+#                          for pytest, e.g., "tox -- --update-goldens")
+# tox -e regexlint       # lint regular expressions with regexlint
+# tox -e pylint          # lint code with pylint
+# tox -e check           # perform various checks, including linting with pyflakes
+# tox -e mapfiles        # regenerate map files
+# tox -e coverage        # run tests and output a coverage report
+# tox -e doc             # compile documentation (you can pass extra arguments for
+#                          for Sphinx, e.g., "tox -e doc -- -b dirhtml" to select the
+#                          dirhtml builder)
+# tox -e web-doc         # compile documentation with demo support (uses Pyodide to
+#                          build Pygments for WASM)
+
 [tox]
-envlist = py{37, 38, 39, 310, 311, 312}, lint
+envlist = py
```

Comment:
That's fine because we run it in a matrix in CI anyways, and locally nobody has all the versions installed. For me it always skipped all but one version for example.